test/core/rules/test_cases/L030.yml

```diff
@@ -0,0 +1,11 @@
+rule: L030
+test_1:
+  fail_str: SELECT MAX(id), min(id) from table
+  fix_str: SELECT MAX(id), MIN(id) from table
+test_2:
```

Comment:
Can we put a note in CONTRIBUTING to ask contributors to please use descriptive names for test identifiers ? for example:

`fix_inconsistent_casing`

Might need to explain that "old" tests were implemented before the YAML-based rule tests and so their identifiers aren't descriptive

Reply:
This relates to the test description @pwildenhain was talking about, which would also be nice IMO

Reply:
Ah I didn't even think about just naming the yaml key differently, I 🤔 think I like that better, and maybe we still have an optional key like `issue:` which gives more detail behind the test case?

Reply:
Good call. I'm going to write a small readme in the test folder to explain how the syntax works